libraries/AP_Follow/AP_Follow.h

```diff
@@ -123,8 +131,11 @@ class AP_Follow
     uint32_t _last_heading_update_ms;   // system time of last heading update
     float _target_heading;          // heading in degrees
     bool _automatic_sysid;          // did we lock onto a sysid automatically?
-    float   _dist_to_target;        // latest distance to target in meters (for reporting purposes)
+    float   _dist_to_target_xy;     // latest distance to target position (with offsets) in meters (for reporting purposes)
+    float   _dist_to_target_z;      // latest distance to target position (with offsets) in meters vertically (for reporting purposes)
     float   _bearing_to_target;     // latest bearing to target in degrees (for reporting purposes)
+    float   _dist_to_vehicle_xy;    // latest distance to target vehicle (for reporting purposes)
```

Comment:
Can you elaborate a bit more on the difference between dist_to_target and dist_to_vehicle ?

Reply:
@amilcarlucas, the dist_to_target includes the offset from the vehicle (held in FOLL_OFS_ parameters)